### PR TITLE
More lenience for COPYING and LICENSE-like files

### PR DESCRIPTION
**Pull request by carmenbianca**
Related to https://github.com/fsfe/reuse-tool/pull/1041

### CODE REVIEW

File: site/content/en/spec-3.3.md [hunk 1]
```diff
@@ -69,7 +69,9 @@ Covered Files are any file which must contain Licensing Information. This is
 equal to all files in a project, with the exception of:
 
 - The License Files stored in the `LICENSES/` directory.
-- `COPYING` and `LICENSE`, with or without file extensions.
+- `COPYING`, `LICENSE`, and `LICENCE`, with or without file extensions. These
+  MAY also contain a dash separator, like `LICENSE-MIT`. These files SHOULD be
```
Comment (silverhook) [inline site/content/en/spec-3.3.md:73]:
```suggestion
  MAY also contain a dash or dot separator, like `LICENSE-MIT` or `COPYING.GPL`. These files SHOULD be
```

I’m still unsure I like this, seems like a project-specific thing. But if we’re doing it, let’s do it proper and also include the dot. Calling `.GPL.markdown` a file extension would still stretching it quite a bit.

File: CHANGELOG.md [hunk 2]
```diff
@@ -44,6 +44,10 @@ This is the change log for the REUSE Specification.
 
 ### Changed
 
+- Also allow `COPYING` and `LICENSE` files to have a suffix with a dash (e.g.
```
Comment (silverhook) [inline CHANGELOG.md:47]:
```suggestion
- Also allow `COPYING` and `LICENSE` files to have a suffix with a dash or dot (e.g. `COPYING.GPL or `
```

**Review by silverhook — CHANGES_REQUESTED** [review]
I’m still not the biggest fan of this and hope it doesn’t open a Pandora’s box, but if we’re doing it, let’s also make sure the dot is included, and not just implied as a file extension.

If nothing else, both are used in SPDX ids, so e.g. `LICENSE-GPL-3.0.txt` does include both dashes and dots (and an extension too)

File: site/content/en/spec-3.3.md (hunk 1 above)
Comment (carmenbianca) [reply site/content/en/spec-3.3.md]: implemented

File: CHANGELOG.md (hunk 2 above)
Comment (carmenbianca) [reply CHANGELOG.md]: implemented

**Pull request merged**